### PR TITLE
NVMe Command constant needs to use full path

### PR DESCRIPTION
**Pull request by karthikk92**
# Description
NVMeCommand constant needs to use full path

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1549 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Image build pointing to gonvme branch and installed powerstore driver and ran basic scale testing

UTs added and now coverage increased from 20% to 21.3%
image (5)

UTS covered under this PR: https://github.com/dell/gonvme/pull/54, once this is merged, will merge present PR.

### CODE REVIEW

File: gonvme_tcp_fc.go [hunk 1]
```diff
@@ -54,17 +54,45 @@ const (
 type NVMe struct {
 	NVMeType
 	sessionParser NVMeSessionParser
+	NVMeCommand string
 }
 
+// func (nvme *NVMe) setNVMePath() {
```
Comment (gallacher) [inline gonvme_tcp_fc.go:60]: Please remove commented code

File: gonvme_tcp_fc.go [hunk 2]
```diff
@@ -54,17 +54,30 @@ const (
...
 		},
 	}
 	nvme.sessionParser = &sessionParser{}
+	if nvme.getChrootDirectory() != "/" {
+		command = append([]string{"chroot", nvme.getChrootDirectory()}, "which", DefaultNVMeCommand)
+	} else {
+		command = []string{"which", DefaultNVMeCommand}
+	}
+	log.Infof("NVMe %s",command)
+	path, err := exec.Command(command[0], command[1:]...).Output()
```
[the remark this answers is not in the document]
Comment (karthikk92) [reply gonvme_tcp_fc.go]: @donatwork as discussed yesterday, we tried checking the different possible paths as its updated in commits, but its not working as expected as it checks the path locally and throws path does not exist.
image

[the remark this answers is not in the document]
Comment (karthikk92) [reply gonvme_tcp_fc.go]: @donatwork  code changes done with list of paths now and updated in the PR.

**Comment by karthikk92** [comment]
Will add Single PR for Both defect fix and UT coverage

**Pull request closed without merging**